projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -187,6 +187,9 @@ protected function __construct() {
 		add_action( 'wp_ajax_grunion-contact-form', array( $this, 'ajax_request' ) );
 		add_action( 'wp_ajax_nopriv_grunion-contact-form', array( $this, 'ajax_request' ) );
 
+		add_action( 'wp_ajax_jetpack_unauth_file_download', array( $this, 'handle_file_download' ) );
```

Comment:
Should we move this to jetpack? Code base somewhere? Instead of this package?

Reply:
yes :)

Reply:
same for the one bellow, probably an `unauth_file_upload` package?